ElCarro.Web/Web.config

```diff
@@ -10,9 +10,10 @@
   </configSections>
   <connectionStrings>
     <!--<add name="DefaultConnection" connectionString="Data Source=(LocalDb)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\aspnet-ElCarro.Web-20160811110821.mdf;Initial Catalog=aspnet-ElCarro.Web-20160811110821;Integrated Security=True" providerName="System.Data.SqlClient" />-->
-    <add name="DefaultConnection" connectionString="Server=tcp:elcarro.database.windows.net,1433;Initial Catalog=elcarroDB;Persist Security Info=False;User ID=elcarro;Password=_Welc0me1_;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;" providerName="System.Data.SqlClient" />
+    <add name="DefaultConnection" connectionString="Data Source=(LocalDb)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\ElCarro.mdf;Initial Catalog=ElCarro;Integrated Security=True" providerName="System.Data.SqlClient" />
```

Comment:
You should not use the connection string of azure or other services in the source code. We are supposed to use some transformation step in the continuous integration system. @eliancruz29